CONTRIBUTING.md

```diff
@@ -0,0 +1,17 @@
+# Issues
```

Comment:
```suggestion
# Contributing to Ion

## Filing Issues
```

Some useful information to include under `Contributing to Ion` would be something along the lines of 
```
Welcome to the [TJ CSL][https://sysadmins.tjhsst.edu) repository for Intranet 3! This document explains all you need to know about contributing to Ion.

If you are a TJHSST student, you can join the `#ion` channel on the [TJ CSL Slack](https://tjcsl.slack.com) to ask questions and interact with other Ion developers.  If you are not a TJHSST student, feel free to send an email to intranet [at] tjhsst [dot] edu and we will get to you as soon as we can.
```